install/kubernetes/istio.yaml

```diff
@@ -1151,7 +1201,7 @@ spec:
         - -v
         - "2"
         - --discoveryAddress
-        - istio-pilot:8080
+        - istio-pilot:15003
```

Comment:
can you remind me why you changed the ports for pilot and mixer?

Reply:
It allows for upgrades to be done without downtime. We need to keep both old and new ports open at the same time.